src/app/core/services/loading-block.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class LoadingBlockService {
+  private notificationSource: Subject<boolean> = new Subject();

```

Comment:
use type of subject you can pass the initial value to

Reply:
fixed